system/Helpers/text_helper.php

```diff
@@ -310,7 +310,7 @@ function convert_accented_characters(string $str): string
         if (! is_array($arrayFrom)) {
             $config = new ForeignCharacters();
 
-            if (empty($config->characterList) || ! is_array($config->characterList)) {
+            if ($config->characterList === [] || ! is_array($config->characterList)) {
```

Comment:
I think the is_array check should come first.

Reply:
It using OR operator, so thats fine, as it verify it value as empty array OR not array, it compare empty array is nothing todo with type

Reply:
I think the is_array check is useless anyway, but since the property is not yet natively typed this should be fine for now. 👍